src/validate.js

```diff
@@ -152,24 +165,44 @@ export default function validateFormData(
   formData,
   schema,
   customValidate,
-  transformErrors
+  transformErrors,
+  additionalMetaSchemas = []
 ) {
+  // add more schemas to validate against
+  if (
+    additionalMetaSchemas &&
+    !deepEquals(formerMetaSchema, additionalMetaSchemas) &&
+    Array.isArray(additionalMetaSchemas)
+  ) {
+    ajv = createAjvInstance();
+    ajv.addMetaSchema(additionalMetaSchemas);
+    formerMetaSchema = additionalMetaSchemas;
+  }
+
+  let validationErrors = null;
   try {
     ajv.validate(schema, formData);
-  } catch (e) {
-    // swallow errors thrown in ajv due to invalid schemas, these
-    // still get displayed
+  } catch (err) {
+    validationErrors = err;
```

Comment:
@aerfio `validationErrors` is an object, not an array, right? Would it make more sense to rename it to `validationError`?